by-language/scala-spark-http/README.md

```diff
@@ -0,0 +1,24 @@
+# Scala example loading data from Spark to CrateDB via http
+
+This example aims to demonstrate how a Spark data frame can be loaded into CrateDB using the CrateDB http endpoint.
+
+It assumes there is a CrateDB instance running on localhost accepting connections with the default `crate` superuser, and it relies on the following table being created:
+
+.. code-block:: sql
+
+    CREATE TABLE myschema.mytable (
+        examplefield1   TIMESTAMP,
+        anotherfield    geo_point,
+        jsonpayload     OBJECT
+    );
+
+In environments with very high volumes of data you may want to consider replacing scalaj.http with async calls with `akka.http` or `AsyncHttpClient`.
+You may also want to explore if connection pooling is useful in your environment, and if JDBC calls leveraging the PostgreSQL wire protocol are more convenient for your particular case.
```

Comment:
Just more suggestions again. Feel free to converge them into a better wording, if you like them.

```suggestion
You may also want to explore if connection pooling is useful in your environment,
and if JDBC calls leveraging the PostgreSQL wire protocol are more convenient
for your particular case.

Saying this, note that this example uses [CrateDB's HTTP bulk operation] to ingest
data, which is currently the most efficient way to do it.

[CrateDB's HTTP bulk operation]: https://cratedb.com/docs/guide/performance/inserts/bulk.html
```